src/components/ReviewList.jsx

```diff
@@ -22,29 +24,49 @@ function ReviewList() {
       })
       .then((res) => {
         setReviews(res.data.reviews);
+      })
+      .catch((err) => {
+        setErr("404 category not found");
+      });
+
+    axios
+      .get(`https://tr-games-api.herokuapp.com/api/categories`)
+      .then((res) => {
+        setCategories(
+          res.data.categories.map((category) => {
+            return category.slug;
+          })
+        );
```

Comment:
consider extracting these api calls to functions in an api.js.

✅ tidier components
✅ reusable code
✅ can switch axios/API easier if it's in one place

Reply:
Going to do this in a separate pull request to avoid confusion